Makefile

```diff
@@ -54,11 +54,11 @@ else ifeq ($(RELEASE), 1)
 CARGO_OSDK_ARGS += --release
 endif
 
-ifeq ($(INTEL_TDX), 1)
+ifeq ($(CVM_GUEST), 1)
```

Comment:
In Makefile, the name of this variable should still be called `INTEL_TDX` because the scheme is TDX-specific.